File: java/URI_1074.java
package URI;
import java.util.Scanner;

public class URI_1074 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);	
		
		int n = sc.nextInt();		
		for(int i = 1; i <= n ; i ++) {			
			int x = sc.nextInt();
			
			if(x < 0) {				
				if(x % 2 == 0) {
					System.out.println("EVEN NEGATIVE");
				}
				else {
					System.out.println("ODD NEGATIVE");
				}
			}			
			else if(x > 0) {
				if(x % 2 == 0) {
					System.out.println("EVEN POSITIVE");
				}
				else {
					System.out.println("ODD POSITIVE");
				}
			}			
			else {
				System.out.println("NULL");
			}
			
		}
		
		sc.close();

	}

}

File: kotlin/URI_1074.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    val n = sc.nextInt()
    repeat(n) {
        val numero = sc.nextInt()
        when {
            numero == 0 -> println("NULL")
            numero % 2 == 0 -> {
                if (numero > 0) println("EVEN POSITIVE")
                else println("EVEN NEGATIVE")
            }
            else -> {
                if (numero > 0) println("ODD POSITIVE")
                else println("ODD NEGATIVE")
            }
        }
    }
}
